modules/sdk-coin-avaxp/src/lib/transaction.ts

```diff
@@ -168,14 +165,15 @@ export class Transaction extends BaseTransaction {
     if (!this.avaxPTransaction) {
       throw new InvalidTransactionError('Empty transaction data');
     }
+    const memo = 'getMemo' in this.avaxPTransaction ? utils.bufferToString(this.avaxPTransaction.getMemo()) : undefined;
```

Comment:
in which case is `getMemo()` not defined?
please add a comment to the code explaining

Reply:
done